src/test/java/com/restassured/exercise/BaseClassAuth.java

```diff
@@ -0,0 +1,54 @@
+package com.restassured.exercise;
+
+import org.testng.annotations.BeforeClass;
+import io.restassured.RestAssured;
+
+import static io.restassured.RestAssured.given;
+
+import java.util.Base64;
+
+import org.json.simple.JSONObject;
+import org.testng.annotations.Test;
+
+import io.restassured.http.ContentType;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+
+public class BaseClassAuth {
+	 
+	
+	@BeforeClass
+	public void setup() {
+		
+		RestAssured.authentication = RestAssured.preemptive().basic("postman", "password");
+		
```

Comment:
delete extra space required

Reply:
Changes made locally and will be loaded in the next PR.